README.md

```diff
@@ -282,7 +283,7 @@ loader.config({ monaco });
 If you use pnpm, you have to add `vscode` / `monaco-vscode-api` as direct dependency (see the [following table](#monaco-editor--monaco-vscode-api-compatibility-table)), otherwise the installation will fail.
 
 ```json
-"vscode": "npm:@codingame/monaco-vscode-api@1.81.7"
+"vscode": "npm:@codingame/monaco-vscode-api@1.82.0"
```

Comment:
1.82.0 failed :x the first valid version is 1.82.1, no change with 1.82.2

Reply:
You must have checked an outdated version. This is `1.82.2`

Reply:
indeed, no idea what happened :shrug: